x-pack/plugins/infra/public/plugin.ts

```diff
@@ -10,21 +10,21 @@ import { AppMountParameters, PluginInitializerContext } from 'kibana/public';
 import { from } from 'rxjs';
 import { map } from 'rxjs/operators';
 import { DEFAULT_APP_CATEGORIES } from '../../../../src/core/public';
-import { createMetricThresholdAlertType } from './alerting/metric_threshold';
 import { createInventoryMetricAlertType } from './alerting/inventory';
-import { getAlertType as getLogsAlertType } from './alerting/log_threshold';
+import { getLogThresholdAlertType } from './alerting/log_threshold';
```

Comment:
For metrics the exported function is called `createMetricThresholdAlertType`. For the sake of naming consistency with metrics let's use getLogThresholdAlertType or createLogThresholdAlertType. Not sure which one is better though :-)

Reply:
:+1: I'll change it to `createLogThresholdAlertType`.